feature-group-definitions/http11.yml

```
@@ -1,4 +1,5 @@
 name: HTTP/1.1
+description: HTTP/1.1 is a revision of the HTTP network protocol used by the World Wide Web.
```

Hmm, this feature might be hard to explain. There was both HTTP 0.9 and 1.0, so this isn't the earliest version of the protocol, but it is the oldest worth describing I think.

Maybe...

```suggestion
description: HTTP/1.1 is a network protocol used by browsers and servers. It has been superseded by HTTP/2 and HTTP/3.
```